lib.php

```diff
@@ -46,6 +46,14 @@ function kialo_supports($feature) {
     }
 }
 
+/**
+ * Prevent the Kialo icon from having its colors modified on Moodle >= 4.4.
+ * For < 4.4 this is handled in style.css.
+ */
+function kialo_is_branded(): bool {
+    return true;
+}
```

Comment:
nice! didn't know about this

Reply:
I think this is technically unnecessary now that we have removed the `FEATURE_MOD_PURPOSE` return value, but I'd just keep it in here as it doesn't hurt either.